calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
```

Comment:
Does the declaration necessitate the `const` and `function` keywords?  

```suggestion
function calculateAverage(grades) {
```

Reply:
testing a reply(as a single comment)